pypowerwall/__main__.py

```diff
@@ -20,7 +20,7 @@
 
 # Global Variables
 AUTHFILE = ".pypowerwall.auth"
-authpath = os.getenv("PW_AUTHPATH", "")
+authpath = os.getenv("PW_AUTH_PATH", "")
```

Comment:
That was it!  The `setup` works now.

Reply:
Using: jasonacox/pypowerwall:0.8.0t51-beta2

Reply:
Yeah, sorry, my bad, I mechanically replaced it as if that was added by me for local testing =(